src/client/config/dev-cloud.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "useEndPoint": true,
```

Comment:
I believe in the case of server deployment. There are cases when we want to use window.location, otherwise we have to update the deployment so we write out config for each deployment we push to